muranotest/config2.conf

```diff
@@ -0,0 +1,56 @@
+[murano]
+# keystone url
+auth_url = http://cloud.lab.fi-ware.org:4731/v3
```

Comment:
Why we need this auth_url with value eq to xxxx.fi-ware.org:xxx when we have the auth_uri with the value xxxx.fiware.org:xxx?